tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -131,12 +131,12 @@ class BaseExampleGenExecutor(
 
   The conversion is done in `GenerateExamplesByBeam` as a Beam pipeline, which
   validates the configuration, reads the external data sources, converts the
-  record in the input source to TF record if needed, and splits the examples if
-  the output split config is given. Then the executor's `Do` writes the results
-  in splits to the output path.
+  record in the input source to tf.Example or tf.SequenceExample if needed,
```

Comment:
nit, given we also allow subclass to support any bytes, consider to reword this to  `input source to any supported output payload formats (e.g., tf.Example or tf.SequenceExample)`

Reply:
got it, this makes sense